source/index.rst

```diff
@@ -2,6 +2,12 @@
 Sponge Documentation
 ====================
 
+.. warning::
+
+    This documentation refers to outdated API version and is no longer actively maintained. While the code examples
```

Comment:
"refers to *an* outdated API version" ?

Reply:
And on the other one too. Though it doesn't really refer to any API version there...

Reply:
Fixed. I don't know how to express the "its outdated part" otherwise.